src/core/dfn-panel.js

```diff
@@ -14,6 +14,13 @@ export async function run() {
     document.querySelector("link")
   );
 
+  /** @type {HTMLElement} */
+  const elems = document.querySelectorAll("dfn, .index-term");
+  for (const el of elems) {
+    const panel = createPanel(el);
+    document.body.append(panel);
```

Comment:
Would it be better to add all panels to a fragment and then append all at once?

Reply:
That might be cleaner.